packages/st2bundle/install_rhel6.md

```diff
@@ -0,0 +1,48 @@
+## Install IUS repo
+sudo yum install -y https://dl.iuscommunity.org/pub/ius/stable/Redhat/6/x86_64/ius-release-1.0-14.ius.el6.noarch.rpm
+
+# Install mongodb
+sudo yum install -y mongodb mongodb-server
+
+# Install rabbitmq
+sudo rpm --import https://www.rabbitmq.com/rabbitmq-signing-key-public.asc
+curl -sS -k -o /tmp/rabbitmq-server.rpm https://www.rabbitmq.com/releases/rabbitmq-server/v3.3.5/rabbitmq-server-3.3.5-1.noarch.rpm
```

Comment:
I suggest to preinstall `wget` and use it instead of curl, usually wget is always available. Since there's no curl installed previously - this is not safe.